crates/configuration/src/relaychain.rs

```diff
@@ -78,6 +79,11 @@ impl RelaychainConfig {
     pub fn nodes(&self) -> Vec<&NodeConfig> {
         self.nodes.iter().collect::<Vec<&NodeConfig>>()
     }
+
+    /// The nodes of the relay chain.
```

Comment:
This code is not exposed publicly:

```suggestion
    // The nodes of the relay chain.
```

Given how straightforward is this code, we can even skip it ?